my-app/src/firebase.js

```diff
@@ -0,0 +1,15 @@
+import firebase from 'firebase'
```

Comment:
Confusing to have a module named firebase and our own file named firebase

Reply:
@micahcatlin would you look at this - I'm not sure what normal patterns are.

Reply:
I think this is fine.